### DISCUSSION TITLE
Allow us to prefix Directory Data Files with an underscore

### DISCUSSION THREAD

**Discussion by mattwaler** [11ty/eleventy#4037] [Enhancement Queue]
**Is your feature request related to a problem? Please describe.**

In large folders full of lots of MD files, having to mimic the directory name exactly causes the config file to get lost among the MD files. See screenshot:

![Screen Shot 2019-11-16 at 3 19 31 PM](https://user-images.githubusercontent.com/32300026/68998765-8ea36680-0884-11ea-8edc-879f9ec2a506.jpg)

**Describe the solution you'd like**

Allowing us to prefix the Directory Data File with an underscore, it would sort to the top of the folder, allowing for easy identification and separation from the content files. Something like this:

![Screen Shot 2019-11-16 at 3 20 53 PM](https://user-images.githubusercontent.com/32300026/68998781-b5fa3380-0884-11ea-8670-f48ca3099af1.jpg)

**Describe alternatives you've considered**

No other solutions come to mind.

**Additional context**

I love 11ty!

**Comment by mattwaler** [comment]
@zachleat Any possibility for this being implemented? I might be able to take a crack myself!

**Comment by ithinkihaveacat** [comment]
Requiring the `*.11tydata.js` to match the directory name also means that renaming the directory also requires renaming the file as well, which easy to miss and annoyingly to do. Using and underscore and also using the same name (e.g. `_index.11tydata.js` might be better.

**Comment by mattwaler** [comment]
That's not a bad idea @ithinkihaveacat, I didn't think about that.

Maybe a happy medium would be combining the _ and the `jsDataFileSuffix` config value to create this, so the file would be `/dir/_11tydata.js`, and could be customized.

Not sure if that's possible though!

**Comment by zachleat** [comment]
I believe https://github.com/11ty/eleventy/pull/1699 is related to this.

I don’t think we want to add additional magic here but #1699 might be a nice configurable way out!

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!
